static/app/views/ddm/summaryTable.tsx

```diff
@@ -1,47 +1,127 @@
           const {avg, min, max, sum} = getValues(data);
 
           return (
             <Fragment key={seriesName}>
-              <FlexCell onClick={() => onClick(seriesName)} hidden={hidden}>
-                <ColorDot color={color} />
-              </FlexCell>
-              <Cell onClick={() => onClick(seriesName)}>
-                {getNameFromMRI(seriesName)}
-              </Cell>
-              {/* TODO(ddm): Add a tooltip with the full value, don't add on click in case users want to copy the value */}
-              <Cell>{formatMetricsUsingUnitAndOp(avg, unit, operation)}</Cell>
-              <Cell>{formatMetricsUsingUnitAndOp(min, unit, operation)}</Cell>
-              <Cell>{formatMetricsUsingUnitAndOp(max, unit, operation)}</Cell>
-              <Cell>{formatMetricsUsingUnitAndOp(sum, unit, operation)}</Cell>
+              <CellWrapper
+                onClick={() => onClick(seriesName)}
+                onMouseEnter={() => setHoveredLegend?.(seriesName)}
+                onMouseLeave={() => setHoveredLegend?.('')}
+              >
+                <Cell>
+                  <ColorDot color={color} hiddenn={!!hidden} />
+                </Cell>
+                <Cell>{getNameFromMRI(seriesName)}</Cell>
+                {/* TODO(ddm): Add a tooltip with the full value, don't add on click in case users want to copy the value */}
+                <Cell right>{formatMetricsUsingUnitAndOp(avg, unit, operation)}</Cell>
+                <Cell right>{formatMetricsUsingUnitAndOp(min, unit, operation)}</Cell>
+                <Cell right>{formatMetricsUsingUnitAndOp(max, unit, operation)}</Cell>
+                <Cell right>{formatMetricsUsingUnitAndOp(sum, unit, operation)}</Cell>
+              </CellWrapper>
+              {hasActions && (
+                <Cell right>
+                  <ButtonBar gap={0.5}>
+                    {transaction && (
+                      <div>
+                        <Tooltip title={t('Open Transaction Summary')}>
+                          <LinkButton
+                            to={transactionSummaryRouteWithQuery({
+                              orgSlug: slug,
+                              transaction,
+                              projectID: selection.projects.map(p => String(p)),
+                              query: {
+                                query: '',
+                                environment: selection.environments,
+                                start: selection.datetime.start
+                                  ? getUtcDateString(selection.datetime.start)
+                                  : undefined,
+                                end: selection.datetime.end
+                                  ? getUtcDateString(selection.datetime.end)
+                                  : undefined,
+                                statsPeriod: selection.datetime.period,
+                              },
+                            })}
```

Comment:
small nit: maybe calculate this and the other `to` prop in the function body to keep the jsx easy to read?

Reply:
@ArthurKnaus auto-merge kicked in, will apply the feedback in a follow-up 👍